### PR TITLE
zed: update to 0.137.3

### PR DESCRIPTION
**Pull request by ognevny**

### CODE REVIEW

File: mingw-w64-zed/001-download-pc-windows-gnu-assets.patch [hunk 1]
```diff
@@ -23,11 +23,11 @@ index 02231765d..8e1ab4959 100644
 +                zed::Os::Windows => "pc-windows-gnu",
              },
          );
- 
+
 diff --git a/extensions/gleam/src/gleam.rs b/extensions/gleam/src/gleam.rs
 index 8deef729d..bfafaa435 100644
---- a/extensions/gleam/src/gleam.rs
-+++ b/extensions/gleam/src/gleam.rs
+--- a/zed-0.137.2/extensions/gleam/src/gleam.rs
```
Comment (Biswa96) [inline mingw-w64-zed/001-download-pc-windows-gnu-assets.patch:29]: patches should not have source directory in path. Use cd or pushd to change current directory before applying the patch.

Comment (ognevny) [reply mingw-w64-zed/001-download-pc-windows-gnu-assets.patch:29]: I wanted to write all patches into one file. is there a way to do this?

Comment (Biswa96) [reply mingw-w64-zed/001-download-pc-windows-gnu-assets.patch:29]: Is there any reason to do that?

Comment (ognevny) [reply mingw-w64-zed/001-download-pc-windows-gnu-assets.patch:29]: to avoid creating of 3 patch files

Comment (ognevny) [reply mingw-w64-zed/001-download-pc-windows-gnu-assets.patch]: done

**Comment by ognevny** [comment]
changed the installation method to avoid recompilation and for easier renaming

**Pull request merged**